tests/bindinginspectortest.cpp

```diff
@@ -0,0 +1,1009 @@
+    QCOMPARE(aImplicitWidthIndex2.sibling(0, BindingModel::s_depthColumn).data().toString(), QStringLiteral("∞"));
+    QCOMPARE(bindingModel->rowCount(aImplicitWidthIndex2), 1);
+
+    Probe::instance()->selectObject(text);
+    QCOMPARE(bindingModel->rowCount(), 6);
+    QModelIndex tAnchorsVerticalCenterIndex = bindingModel->index(5, 0, QModelIndex());
+    QVERIFY(tAnchorsVerticalCenterIndex.isValid());
+    QCOMPARE(tAnchorsVerticalCenterIndex.data().toString(), "t.anchors.verticalCenter");
+    QCOMPARE(tAnchorsVerticalCenterIndex.sibling(5, BindingModel::s_depthColumn).data().toString(), QStringLiteral("2"));
+    QCOMPARE(bindingModel->rowCount(tAnchorsVerticalCenterIndex), 2); // is `parent` and `parent.verticalCenter`
+
+    QModelIndex aVerticalCenterIndex;
+    QModelIndex aVerticalCenterIndex0 = bindingModel->index(0, 0, tAnchorsVerticalCenterIndex); // It's more or less random which row contains
+    QModelIndex aVerticalCenterIndex1 = bindingModel->index(1, 0, tAnchorsVerticalCenterIndex); // the correct property.
+    QVERIFY((aVerticalCenterIndex0.data().toString() == "a.verticalCenter" && aVerticalCenterIndex1.data().toString() == "t.parent")
+            || (aVerticalCenterIndex1.data().toString() == "a.verticalCenter" && aVerticalCenterIndex0.data().toString() == "t.parent"));
+    if (aVerticalCenterIndex0.data().toString() == "a.verticalCenter") {
+        aVerticalCenterIndex = aVerticalCenterIndex0;
+        QCOMPARE(aVerticalCenterIndex.sibling(0, BindingModel::s_depthColumn).data().toString(), QStringLiteral("1"));
+        QCOMPARE(bindingModel->rowCount(aVerticalCenterIndex), 2);
+    } else {
+        aVerticalCenterIndex = aVerticalCenterIndex1;
+        QCOMPARE(aVerticalCenterIndex.sibling(1, BindingModel::s_depthColumn).data().toString(), QStringLiteral("1"));
+        QCOMPARE(bindingModel->rowCount(aVerticalCenterIndex), 2);
+    }
+    QVERIFY(aVerticalCenterIndex.isValid());
+
+    QModelIndex aYIndex = bindingModel->index(0, 0, aVerticalCenterIndex);
+    QVERIFY(aYIndex.isValid());
+    QCOMPARE(aYIndex.data().toString(), "a.y");
+    QCOMPARE(aYIndex.sibling(0, BindingModel::s_valueColumn).data().toString(), QStringLiteral("0"));
+    QCOMPARE(aYIndex.sibling(0, BindingModel::s_depthColumn).data().toString(), QStringLiteral("0"));
+    QCOMPARE(bindingModel->rowCount(aYIndex), 0);
+
+    QModelIndex aHeightIndex = bindingModel->index(1, 0, aVerticalCenterIndex);
+    QVERIFY(aHeightIndex.isValid());
+    QCOMPARE(aHeightIndex.data().toString(), "a.height");
+    QCOMPARE(aHeightIndex.sibling(1, BindingModel::s_valueColumn).data().toString(), QStringLiteral("200"));
+    QCOMPARE(aHeightIndex.sibling(1, BindingModel::s_depthColumn).data().toString(), QStringLiteral("0"));
+    QCOMPARE(bindingModel->rowCount(aHeightIndex), 0);
+
+    Probe::instance()->selectObject(nullptr); //TODO: is this the correct way (seems to crash in an independent part of GammaRay without)
```

Comment:
Should not crash, still it is good to test this case and ensure your binding model is now empty / resetted